src/main/java/cloud/eppo/EppoHttpClient.java

```diff
@@ -0,0 +1,116 @@
+package cloud.eppo;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.HttpUrl;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class EppoHttpClient {
```

Comment:
Adapted from the Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/main/java/cloud/eppo/android/EppoHttpClient.java)) but uses OkHtttp3 and has a synchronous method.